allow flowr-frontend to toggle the keyboard

lmancel opened the pull request:
also support both external and internal controls
either http call or flowr-desktop's managed

Commit:
allow flowr-frontend to toggle the keyboard

also support both external and internal controls
either http call or flowr-desktop's managed

Alexandre1624 commented on src/frontend/preloads/exportNode.ts:
```diff
@@ -1,13 +1,15 @@
-import type { IpcRenderer } from 'electron'
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+import { IpcRenderer } from 'electron'
 import { debounce } from 'lodash'
```
`import debounce from 'lodash/debounce`

lmancel replied:
Does not work with the webpack config here

Commit:
misc: lodash import

Commit:
Merge remote-tracking branch 'origin/bugfix/FLOW-10781-keyboard' into bugfix/FLOW-10781-chromium-nav-crash

Commit:
Update Electron to 19.0.10 + update ffmpeg-static and ffprobe-static

Electron update fixes "Network service crashes issues". ff* updates allows to run the project on Mac M1 laptops.

Commit:
misc(ffmpeg) comment for frag duration

Commit:
Merge remote-tracking branch 'origin/bugfix/FLOW-10781-keyboard' into bugfix/FLOW-10781-chromium-nav-crash

phil1618 committed:
Merge pull request #52 from taktik/bugfix/FLOW-10781-chromium-nav-crash

Fix Chromium crash on navigation without refresh

Commit:
misc(preload) revert lodash import

default "import" mixed with commonjs "require" fucks up module resolution
Need to spend a bit more time investigating it
Or maybe completely rework this preload script to make it cleaner

Commit:
remove ffprobe from build

Not used anymore; will save around 200Mb
